site/_docs/configuration.md

```diff
@@ -32,7 +32,8 @@ class="flag">flags</code> (specified on the command-line) that control them.
     <tr class="setting">
       <td>
         <p class="name"><strong>Site Source</strong></p>
-        <p class="description">Change the directory where Jekyll will read files</p>
+        <p class="description">Change the directory where Jekyll will
+        read files</p>
```

Comment:
Yeah, I need to look more closely at the tables, code blocks, and other things inside of `<pre>` tags. My original intent was just to get everything under 80 characters for each line, but I can see there is more work that needs to be done.